src/scss/themes/_common-themes.scss

```diff
@@ -1,16 +1,34 @@
-/* Notes: 
+$themes: (
+  'cookbook': (
+    'primary': rgb(51, 134, 212),
+    'on-primary': #fff,
+  ),
+  'cake': (
+    'primary': rgb(199, 159, 98),
+    'on-primary': #fff,
+  ),
+  'lollipop': (
+    surface: false,
+  )
+);
+
+@function get-color($map, $variant) {
+  $value: map-get($map, $variant);
+  @if $value != null {
+    @return $value;
+  }
+  @return null;
+}
 
+/* Notes: 
+Domain idea:
 .domain--jb {
   .theme--cake {
     --surface: rgb(64, 64, 64);
     --on-surface: rgba(255, 255, 255, 0.87);
   }
 }
-
-.domain--jb {
-  color: #fff;
-  
-}
+This wouldn't require the css to be build XX number of times.
```

Comment:
Add more commenting about this, make it very clear that this Tommy magic fixes a performance problem and that is why we do this and that isnt allowed...